docs/docs/quick-start.md

```diff
@@ -48,7 +48,7 @@ Gatsby will perform an optimized production build for your site, generating stat
 npm run serve
 ```
 
-Gatsby starts a local HTML server for testing your built site.
+Gatsby starts a local HTML server for testing your built site. You must use `npm run build` before using this command.
```

Comment:
```suggestion
Gatsby starts a local HTML server for testing your built site. Remember to build your site using  `npm run build` before using this command.
```

Reply:
@sidharthachatterjee I think your suggestion reads nicer 👍

Reply:
@andrecalvo I think you forgot to hit commit suggestion 😂

Reply:
@sidharthachatterjee oops! Apologies

Reply:
Haha, no worries. Thank you for the contribution! Merging.